### PR TITLE
Re-enable java/lang/System/Versions.java

### PR DESCRIPTION
**Pull request by pdbain-ibm**
Fixed under https://github.com/eclipse/openj9/issues/1282

Signed-off-by: Peter Bain <pdbain@ca.ibm.com>

### CODE REVIEW

**Comment by pdbain-ibm** [comment]
@llxia @ShelleyLambert for your review.

File: openjdk_regression/ProblemList_openjdk8-openj9.txt [hunk 1]
```diff
@@ -20,7 +20,7 @@ java/lang/String/CaseInsensitiveComparator.java		https://github.com/eclipse/open
 java/lang/StringBuffer/Exceptions.java		https://github.com/eclipse/openj9/issues/1128	generic-all
 java/lang/StringBuffer/TestSynchronization.java		https://github.com/eclipse/openj9/issues/1128	generic-all
 java/lang/StringBuilder/Exceptions.java		https://github.com/eclipse/openj9/issues/1128	generic-all
-java/lang/System/Versions.java		https://github.com/eclipse/openj9/issues/1128	generic-all
+# java/lang/System/Versions.java		https://github.com/eclipse/openj9/issues/1128	generic-all fixed
```
Comment (smlambert) [inline openjdk_regression/ProblemList_openjdk8-openj9.txt]: Rather than commenting out the line, please remove the line (same comment for both files).

**Comment by smlambert** [comment]
personal build to confirm change is in nightly build: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/test_personal/452/

**Comment by smlambert** [comment]
https://ci.adoptopenjdk.net/view/work%20in%20progress/job/test_personal/453/

Still see failure, appears our 'nightlies' are stale:

16:04:36 openjdk version "1.8.0_162"
16:04:36 OpenJDK Runtime Environment (build 1.8.0_162-b12)
16:04:36 Eclipse OpenJ9 VM (build master-712c2ee2, JRE 1.8.0 Linux amd64-64 Compressed References **20180322_127** (JIT enabled, AOT enabled)
16:04:36 OpenJ9   - 712c2ee2
16:04:36 OMR      - b623c98f
16:04:36 JCL      - 7580d499 based on jdk8u162-b12)

Need to check with @sxa or build team as to why stale nightly build, as I understood those pipelines are scheduled for nightly.

File: openjdk_regression/ProblemList_openjdk8-openj9.txt (hunk 1 above)
Comment (pdbain-ibm) [reply openjdk_regression/ProblemList_openjdk8-openj9.txt]: Thanks.  Done.

**Comment by pdbain-ibm** [comment]
Do you need any changes to this or can it be merged?
Thanks

**Comment by smlambert** [comment]
No further changes required to this PR.
It is gated by some pending build pipeline changes and will be merged when they are committed.

**Pull request merged**